example_scripts/login_system-cache.pwn

```diff
@@ -46,224 +39,286 @@ new Player[MAX_PLAYERS][E_PLAYERS];
 new g_MysqlRaceCheck[MAX_PLAYERS];
 
 //dialog data
-enum 
+enum
 {
-	DIALOG_INVALID,
 	DIALOG_UNUSED,
-	
+
 	DIALOG_LOGIN,
-	DIALOG_REGISTER,
+	DIALOG_REGISTER
 };
 
-
-/*
- * SA-MP callbacks
- */
+main() {}
 
 public OnGameModeInit()
 {
-	mysql_log(LOG_ERROR | LOG_WARNING, LOG_TYPE_HTML); //logs errors and warnings into a nice .html log file
-	g_SQL = mysql_connect(SQL_HOST, SQL_USER, SQL_DB, SQL_PASS);
+	new MySQLOpt: option_id = mysql_init_options();
+
+	mysql_set_option(option_id, AUTO_RECONNECT, true); // It automatically reconnects when loosing connection to mysql server
+	mysql_set_option(option_id, MULTI_STATEMENTS, true); // Allows to execute multiple queries at once
```

Comment:
This "multi-statements" option is related to #39 ? If yes: SQL Injection is the only downside ? Performance is still the same ? Maybe it would be better to enable multi-statements per-query ?